### ISSUE TITLE
Is the feature reconstruction loss used in training?

### ISSUE THREAD

**Issue by anakita** [nianticlabs/monodepth2#233]
Hi,
I came across the following in the supplementary material of the paper.

> As a result, we used a feature reconstruction loss in the appearance matching term, as in [52, 58, 73], by computing the L1 distance on the reprojected and normalized relu1_1 features from an ImageNet pretrained VGG16 [57] as our pe function.  This takes significantly longer to train, but results in qualitatively better depth maps on this dataset.

But I did not find this feature reconstruction loss in the code.Is it not being implemented ?
Else could you please provide a source to refer to and incorporate the same in this code.I tried depth_vo_feat,but I was not able to convert the caffe part to torch.
I am sorry if I have missed out on the loss implementation.
Thanks

**Comment by mrharicot** [comment]
Hi,
The feature reconstruction loss was just used for a single qualitative experiment and we did not release the code for it.
Please see https://github.com/nianticlabs/monodepth2/issues/227 for more details.
I hope this helps!
